src/evmTest/resources/solidity/MinimumViableToken.sol

```diff
@@ -0,0 +1,19 @@
+pragma solidity ^0.4.9;
+
+contract MinimumViableToken {
+    /* This creates an array with all balances */
+    mapping (address => uint256) public balanceOf;
+
+    /* Initializes contract with initial supply tokens to the creator of the contract */
+    function MinimumViableToken(uint256 initialSupply) {
+        balanceOf[msg.sender] = initialSupply;              // Give the creator all initial tokens
+    }
+
+    /* Send coins */
+    function transfer(address _to, uint256 _value) {
+        if (balanceOf[msg.sender] < _value) throw;           // Check if the sender has enough
+        if (balanceOf[_to] + _value < balanceOf[_to]) throw; // Check for overflows
+        balanceOf[msg.sender] -= _value;                     // Subtract from the sender
+        balanceOf[_to] += _value;                            // Add the same to the recipient
```

Comment:
@rtkaczyk This contract is straight from ethereum webpage: https://www.ethereum.org/token#minimum-viable-token ;)